src/interface/interface.json

```diff
@@ -1376,26 +1376,55 @@
           "partials": {
             "Help": {
               "defaults": {
-                "link": "{String} link behind the Help icon",
                 "title": "{String} the title of the Help icon to be shown on hover"
               },
               "methods": {
+                "open": {
+                  "desc": "Open the help flyout"
+                },
+                "close": {
+                  "desc": "Close the help flyout"
+                },
                 "onClick": {
                   "desc": "Calls the provided callback when user clicks on the Help",
                   "params": "HIG.abstract.EventObject.params",
                   "returns": "HIG.abstract.EventObject.returns"
                 },
-                "setLink": {
-                  "desc": "sets the link of the Help",
+                "onClickOutside": {
+                  "desc": "Calls the provided callback when user clicks outside of the Help",
+                  "params": "HIG.abstract.EventObject.params",
+                  "returns": "HIG.abstract.EventObject.returns"
+                },
+                "addGroup": {
+                  "desc": "adds a group to the help menu",
                   "params": [
-                    "{String} link - new link"
+                    "{Group} group - the group to add"
                   ]
                 },
                 "setTitle": {
-                  "desc": "sets the hover title of the Help",
-                  "params": [
-                    "{String} title - new hover title"
-                  ]
+                   "desc": "sets the hover title of the Help",
+                   "params": [
+                     "{String} title - new hover title"
+                   ]
+                 }
+              },
+              "partials": {
+                "Group": {
+                  "methods": {
+                    "addOption": {
+                      "desc": "Adds an option to the group",
+                      "params": "{Option} option - the option to add"
+                    }
+                  },
+                  "defaults": {
+                    "name": "{String} - the name of the group"
```

Comment:
Do the groups really name names?

Reply:
Correct this is never visible, removed in 57c97a78